src/services/safe/safe.service.impl.ts

```diff
@@ -128,6 +133,10 @@ export class SafeServiceImpl extends Service implements SafeService {
         safeId,
         accountService.user.did,
       )
+
+      if(reconstruct.data){
```

Comment:
`reconstructSafe` returns true even when it is in RECOVERING stage. So we need to query the stage of the safe before sending the notification.